Documentation/installation/k8s-install-migration.rst

```diff
@@ -190,7 +190,7 @@ Preparation
 
     .. code-block:: shell-session
 
-        $ cilium install --helm-values values-migration.yaml --helm-auto-gen-values values-initial.yaml
+        $ cilium install -f values-migration.yaml --dry-run-helm-values > values-initial.yaml
```

Comment:
```suggestion
        $ cilium install --values values-migration.yaml --dry-run-helm-values > values-initial.yaml
```

Nit. I personally slightly prefer the long form, as a bit more self-explanatory (if accepted, the same would apply to the other occurrences). WDYT?

Reply:
updated 📝